helm_deploy/values-dev.yaml

```diff
@@ -23,11 +23,7 @@ generic-service:
     TOKEN_VERIFICATION_API_URL: "https://token-verification-api-dev.prison.service.justice.gov.uk"
     PHASE_NAME: "DEV"
     FEATURE_HIDE_DAYS_COLUMNS_IN_INCENTIVES_TABLE: "true"
-    FEATURE_SHOW_ANALYTICS: "true"
-    FEATURE_SHOW_PC_ANALYTICS: "true"
     FEATURE_SHOW_ANALYTICS_TRENDS: "true"
-    PRISONS_WITH_ANALYTICS: "BWI,MDI,WRI"  # sample data only exists for these 3
```

Comment:
this reminds me that data is only available in these 3 prisons locally, on `dev` and `preprod`. removing this feature gate will show a hopefully-useful message rather than a 404

Reply:
Don't know if it's worth doing anything special about it or not 🤔 .

AFAICT we'd get the same error message saying data is not available for that prison. Maybe that's enough?